VotingApplication/VotingApplication.Web/Scripts/Controllers/BasicVoteController.js

```diff
@@ -29,11 +29,12 @@
 
         function activate() {
             $scope.$watch('poll', function () {
-                $scope.options = $scope.poll ? $scope.poll.Options : [];
+                if ($scope.poll) {
+                    $scope.options = $scope.poll.Options;
```

Comment:
A lot of these are doing the same thing. We should pull them up to the VotingPageController in the PollService.getPoll call, and have an extra call for configuring with the poll object, similar to how getVotes.

I think the only one that needs to do additional config is the points vote setting all option.voteValue = 0